vue/router.py

```diff
@@ -2,15 +2,28 @@
 from .factory import Wrapper, VueRouterFactory
 
 
+def init_dict(component):
+    return component.init_dict()
+
+
 class VueRouter(Wrapper):
     @classmethod
     def init_dict(cls):
         return VueRouterFactory.get_item(cls)
 
-    def __new__(cls):
-        return window.VueRouter.new(cls.init_dict())
+    def __new__(cls, VueRouterObject=window.VueRouter):
```

Comment:
can you provide me some links to documentation how to use custom vue-router objects?
Just for me to understand how this works

Reply:
https://ionicframework.com/blog/announcing-the-ionic-vue-beta/

Reply:
thanks, didn't now that there are different router implementations
can you stick with PEP-8 regarding naming? https://www.python.org/dev/peps/pep-0008/#function-and-variable-names
`VueRouterObject` -> `vue_router_class`
I would suggest to call it class, since it is not really a instance of the router if I get this correct?